Sources/Publish/API/DeploymentMethod.swift

```diff
@@ -59,6 +59,11 @@ public extension DeploymentMethod {
                     at: folder.path
                 )
 
+                try shellOut(
+                    to: "git checkout \(branch) || git checkout -b \(branch)",
+                    at: folder.path
+                )
```

Comment:
We want to switch to the specified branch if it exists, or create it if it doesn't.